docs/yaml/functions/vcs_tag.yaml

```diff
@@ -22,7 +22,7 @@ description: |
 
 kwargs:
   command:
-    type: list[str]
+    type: list[exe | external_program | custom_tgt | file | str]
```

Comment:
The `description` field must specify that e.g.

`*(since 0.63.0)* [[@custom_tgt]], [[@exe]], [[@external_program]], and [[@file]] are accepted.`